Lib/test/test_subprocess.py

```diff
@@ -843,8 +843,12 @@ def test_one_environment_variable(self):
         if sys.platform == "win32":
             cmd = ["CMD", "/c", "SET", "fruit"]
         with subprocess.Popen(cmd, stdout=subprocess.PIPE, env=newenv) as p:
```

Comment:
```suggestion
        with subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=newenv) as p:
```

My bad, also needs this added. Although it may be reasonable to do `stderr=subprocess.STDOUT` and then only look at `stdout` below - we shouldn't ever have error output in the successful case, so while it'll cause a failure there, it'll be a "good" failure.